cumulusci/tasks/tests/test_create_package_version.py

```diff
@@ -153,6 +158,18 @@ def mock_get_static_dependencies():
 
 
 class TestPackageConfig:
+    def test_org_config(self, project_config, persistent_org_config):
+        with pytest.raises(
+            TaskOptionsError,
+            match="Org doesn't contain a config_file. It's a persistent org like Devhub or Developer Edition org",
```

Comment:
Consider moving the error message to a constant (see [here](https://github.com/SFDO-Tooling/CumulusCI/blob/520b51128c7702a374da9a8838f38890082fd3de/cumulusci/tasks/marketing_cloud/deploy.py#L37)) and importing it ([example](https://github.com/SFDO-Tooling/CumulusCI/blob/520b51128c7702a374da9a8838f38890082fd3de/cumulusci/tasks/marketing_cloud/tests/test_deploy.py#L18)). This makes it easier to change in one place and helps keep the code clean and simple.